packages/design-system/src/components/WIP/ErrorState/ErrorState.tsx

```diff
@@ -30,7 +30,7 @@ function ErrorState({ title, description, action, link }: ErrorStatePropTypes) {
 
 				{action && <ButtonPrimary {...action} />}
 
-				{link && (typeof link === 'object' ? <Link {...(link as LinkProps)} /> : link)}
+				{link && (React.isValidElement(link) ? link : <Link {...(link as LinkProps)} />)}
```

Comment:
```suggestion
				{link && (isValidElement(link) ? link : <Link {...(link as LinkProps)} />)}
```
We try to be explicit in our React imports, we found that using `React.forwardRef` vs `forwardRef` didn't have the same effect in storybook, so for good mesure...